docs/modules/reference/pages/service-assurance/monitors/IcmpMonitor.adoc

```diff
@@ -51,34 +51,48 @@ This monitor implements the <<reference:service-assurance/introduction.adoc#ref-
 Examples use CentOS/RHEL path names.
 For Debian/Ubuntu, use `/var/lib/opennms/rrd/response`.
 
-Note that you must include the `monitor` section for each service in your definition.
-
 [source, xml]
 ----
 <service name="ICMP" interval="300000" user-defined="false" status="on">
-  <parameter key="retry" value="2"/>
-  <parameter key="timeout" value="3000"/>
-  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/>
-  <parameter key="rrd-base-name" value="icmp"/>
-  <parameter key="ds-name" value="icmp"/>
+  <parameter key="retry" value="2"/> <1>
+  <parameter key="timeout" value="3000"/> <2>
+  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/> <3>
+  <parameter key="rrd-base-name" value="icmp"/> <4>
+  <parameter key="ds-name" value="icmp"/> <5>
 </service>
 
-<monitor service="ICMP" class-name="org.opennms.netmgt.poller.monitors.IcmpMonitor"/>
+<monitor service="ICMP" class-name="org.opennms.netmgt.poller.monitors.IcmpMonitor"/> <6>
 ----
+<1> Number of attempts to test a service’s status
+<2> Timeout for the isReachable method, in milliseconds.
+<3> Base directory of an RRD repository in which to store this service monitor’s response-time samples.
+<4> The file name of the RRD file (minus the .rrd or .jrb file extension)
+<5> Name of the RRD data source (DS) name in which to store this service monitor’s response-time samples.
+<6> Required *monitor* section.
 
 [source, xml]
 ----
 <!-- Advanced example: set DSCP bits and send a large packet with allow-fragmentation=false -->
 <service name="ICMP" interval="300000" user-defined="false" status="on">
-  <parameter key="retry" value="2"/>
-  <parameter key="timeout" value="3000"/>
-  <parameter key="dscp" value="0x1C"/> <!-- AF32: Class 3, Medium drop probability -->
-  <parameter key="allow-fragmentation" value="false"/>
-  <parameter key="packet-size" value="2048"/>
-  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/>
-  <parameter key="rrd-base-name" value="icmp"/>
-  <parameter key="ds-name" value="icmp"/>
+  <parameter key="retry" value="2"/> <1>
+  <parameter key="timeout" value="3000"/> <2>
+  <parameter key="dscp" value="0x1C"/> <!-- AF32: Class 3, Medium drop probability --> <3>
+  <parameter key="allow-fragmentation" value="false"/> <4>
+  <parameter key="packet-size" value="2048"/> <5>
+  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/> <6>
+  <parameter key="rrd-base-name" value="icmp"/> <7>
+  <parameter key="ds-name" value="icmp"/> <8>
 </service>
 
-<monitor service="ICMP" class-name="org.opennms.netmgt.poller.monitors.IcmpMonitor"/>
+<monitor service="ICMP" class-name="org.opennms.netmgt.poller.monitors.IcmpMonitor"/> <9>
 ----
+<1> Number of attempts to test a service’s status
+<2> Timeout for the isReachable method, in milliseconds.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.